app/models/job.rb

```diff
@@ -0,0 +1,10 @@
+class Job < ApplicationRecord
```

Comment:
Does having this in the models section of the code base make it clear enough that this is separate from Sidekiq Jobs?

Reply:
Are you sure you don't want JobJobs?

Reply:
Now that you mention it, maybe `WorkJobber`.